test/indexer/test_hca_indexer.py

```diff
@@ -615,10 +615,24 @@ def test_paired_end_field(self):
                 self.assertEqual(0, len(contents['protocols']))
             else:
                 if aggregate:
-                    self.assertEqual(one(contents['protocols'])['paired_end'], [True])
+                    self.assertEqual(set(one(contents['protocols'])['paired_end']), set([True]))
                 else:
-                    values = [p['paired_end'] for p in contents['protocols'] if 'paired_end' in p]
-                    self.assertEqual(set(values), set([True]))
+                    paired_end_values = [p['paired_end'] for p in contents['protocols'] if 'paired_end' in p]
+                    self.assertEqual(set(paired_end_values), set([True]))
```

Comment:
```suggestion
                    self.assertEqual(paired_end_values, {True, None}))
```

Aside from the use of set literals/comprehensions, my version is also a stronger assertion.